src/evaluate.jl

```diff
@@ -301,19 +291,11 @@ function evaluate(a::TaylorN{Taylor1{T}}, vals::NTuple{N, Taylor1{T}}) where
 
     @assert N == get_numvars()
 
-    a_length = length(a)
     ord = maximum( get_order.(vals) )
     suma = Taylor1(zeros(T, ord))
 
-    for homPol in 1:length(a)
-        for (i, a_coeff) in enumerate(a.coeffs[homPol].coeffs)
-            iszero(a_coeff) && continue
-            tmp = vals[1]^(coeff_table[homPol][i][1])
-            for n in 2:N
-                tmp *= vals[n]^(coeff_table[homPol][i][n])
-            end
-            suma += a_coeff * tmp
-        end
+    for homPol in length(a):-1:1
+        suma += evaluate(a.coeffs[homPol], vals)
```

Comment:
There seem to be several functions with the same content. Could they be unified?

Reply:
You are right, and that is something I have already considered. I propose to open a new issue on that, and leave it for another PR.